openforcefield/tests/test_forcefield.py

```diff
@@ -3497,10 +3590,16 @@ def test_fractional_bondorder_calculated_rdkit(
             if isinstance(force, openmm.PeriodicTorsionForce)
         ][0]
 
+        off_bond_force = [
+            force
+            for force in omm_system.getForces()
+            if isinstance(force, openmm.HarmonicBondForce)
+        ][0]
+
         ret_mol = list(ret_top.reference_molecules)[0]
         bond = ret_mol.get_bond_between(*central_atoms)
 
-        # ambertools appears to yield around 1.0009 for this bond
+        # ambertools appears to yield around 0.99529355 for this bond
```

Comment:
I don't know why, but all day I have been getting the new value (0.99529355) on my machine. On a Linux cluster, it's 1.0009. Relevant snippet:


```
from openforcefield.tests.test_forcefield import create_ethanol
mol = create_ethanol()
from openforcefield.utils.toolkits import *
AmberToolsToolkitWrapper().assign_fractional_bond_orders(mol)
mol.get_bond_between(1, 2).fractional_bond_order
```